robots/suction_hand.xacro

```diff
@@ -22,6 +22,18 @@
           <box size="0.25 0.13 0.05"/>
         </geometry>
       </visual>
+      <collision>
```

Comment:
This isn't needed as it will default to using the visual one.

Reply:
It is necessary to have the collision slightly shorter than the visual here, as the grasp depth means that with collision=visual, all grasps will collide with any other boxes at the same level as the target box.  Also, even though Moveit overrides the collision, the rviz plugin for displaying RobotState/PlanningScene does not, which can cause bugs to be pretty confusing.